modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -40,7 +40,7 @@ interface XHR extends Observable<XHREventMap> {
  * @class tinymce.util.XHR
  * @mixes tinymce.util.Observable
  * @static
- * @summary XHR has been deprecated and marked for removal in TinyMCE 6.0.
+ * @summary XHR has been deprecated in TinyMCE 5.10 and has been marked for removal in TinyMCE 6.0.
```

Comment:
Same here, should this be in `@summary`?

Reply:
Yes, see https://github.com/tinymce/tinymce/pull/7239. This was something requested by @tylerkelly13 because of the weird way Moxiedoc works.

Reply:
Basically as I understand it, if it's a class that's been deprecated it needs to be in `@summary` instead of being in the description.

Reply:
That's right. Any text in the description of a class goes in the heading, so it can't be in there.

Reply:
> because of the weird way Moxiedoc works.

because of the weird way we use the Moxiedoc output :)